extension/background/intentRunner.js

```diff
@@ -525,32 +525,27 @@ registerHandler("getRegisteredNicknames", getRegisteredNicknames);
 
 let pageNames = {};
 
+export async function getRegisteredPageName() {
+  const result = await browser.storage.sync.get("pageNames");
+  pageNames = result.pageNames;
+
+  return pageNames;
+}
+
 export async function registerPageName(name, { url }) {
   name = name.toLowerCase();
   const creationDate = Date.now();
-  const result = await browser.storage.sync.get("pageNames");
-  pageNames = result.pageNames || {};
+  pageNames = getRegisteredPageName() || {};
```

Comment:
We probably won't need this any longer. Instead we can create a new intent that just matches the phrase:

```
[name:pageName]
```

And the new intent should look up and open the page.